src/pacti/iocontract/iocontract.py

```diff
@@ -261,15 +266,23 @@ def elim_vars_by_refining(self: TermList_t, context: TermList_t, vars_to_elim: L
                 List of context terms that will be used to refine the TermList.
             vars_to_elim:
                 Variables to be eliminated.
+            simplify:
+                Whether to simplify self in the given context before variable elimination
+            tactics_order:
+                The order of the tactics to use when transforming each term in the list.
 
         Returns:
-            A list of terms not containing any variables in `vars_to_elim`
-                and which, in the context provided, imply the terms contained in the
-                calling termlist.
+            A tuple consisting of:
```

Comment:
We need to check if mkdocs will pick this docstring correctly. Newlines need additional indentation for them to show up as part of the same structure. Since you have "-", I am not sure what will happen.

Reply:
How do you generate the doc?

Reply:
The docs are generated from the pacti-docs repo.
We can look at the documentation locally via the command `make docs-serve`